### ISSUE TITLE
Implements visualization widget

### ISSUE THREAD

**Issue by atuleu** [formicidae-tracker/studio#8]
Adds a way to visualize the video saved alongside the tracking data, and display an overlay on top of it to display ant position, orientation, identification, trajectories and interactions ( collision detection ).

**Comment by atuleu** [comment]
Maybe wait for #21 to be close before proceeding

**Comment by atuleu** [comment]
@matthiasruegg   there are some big updates coming and maybe wait for #23  to be closed before starting implementation .

**Comment by AdrianoWanderlingh** [comment]
mismatch between the visualisation widget timestamp reported near the time-bar and the actual time-stamp.
![mismatch](https://user-images.githubusercontent.com/47888790/79880790-5f22f800-83e8-11ea-97cc-6efe1303f5ad.png)

**Comment by AdrianoWanderlingh** [comment]
Request for visualisation widget hotkeys: start/stop video, scroll frame by frame (back and forth), go to specific timestamp.

**Comment by atuleu** [comment]
> mismatch between the visualisation widget timestamp reported near the time-bar and the actual time-stamp.
> ![mismatch](https://user-images.githubusercontent.com/47888790/79880790-5f22f800-83e8-11ea-97cc-6efe1303f5ad.png)

Actually thats exactly the same timestamp. The timestamp displayed in the video frame is truncated to the second. The actual frame timstamp also contains microsecond data, that are quite useful when you want to compute speed of an ant. Just use the copy-pasted value.

Also, please open new issues to report anything new.

**Comment by atuleu** [comment]
> Request for visualisation widget hotkeys: start/stop video, scroll frame by frame (back and forth), go to specific timestamp.

Please open a new issue
